### PR TITLE
Task s4 merge

### PR DESCRIPTION
**Pull request by kyam-mayden**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -18,24 +20,26 @@
     <link rel="stylesheet" type="text/css" href="src/assets/css/vendor/bootstrap.min.css">
     <link rel="stylesheet" type="text/css" href="src/assets/css/styles.css">
 </head>
-
 <body>
 <section class="row banner">
-    <div class="home col-xs-3 col-sm-2">
-        <a href="index.php">
+    <div class="home col-xs-4 col-sm-2">
+        <a href="#">
```
Comment (obwansan) [inline index.php]: Has this href been added?

File: src/classes/Product.php [hunk 2]
```diff
@@ -2,13 +2,56 @@
...
+    public function getAvailableColors():string
+    {
+        return $this->availableColors;
+    }
+
+    public function getImageFilePath():string
+    {
+        return $this->imageFilePath;
+    }
+
```
Comment (obwansan) [inline src/classes/Product.php]: white space

**Review by obwansan — COMMENTED** [review]
just one comment

File: index.php (hunk 1 above)
Comment (kyam-mayden) [reply index.php]: Resolved

File: src/classes/Product.php (hunk 2 above)
Comment (kyam-mayden) [reply src/classes/Product.php]: Resolved

**Review by obwansan — APPROVED** [review]

**Pull request merged**